lib/bundles/node-all.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
     extends: [
         '../env/node.js',
         '../rules/all.js',
-        '../overrides/test-max-statements.js'
+        '../overrides/test-max-statements.js',
+        '../overrides/test-max-lines-per-function.js'
```

Comment:
i'll add docs for this later on, but I need to go ahead and get this merged so that I can fix and land #7